File: Contests/Contest861/Q2.py
def sumPairs(arr, n):
     
    sum = 0
    for i in range(n - 1, -1, -1):
        sum += i*arr[i] - (n-1-i) * arr[i]
    return sum


num_of_test = int(input())

for _ in range(num_of_test):
    n , x = map(int,input().split())
    
    s=0
    m=[]
    for i in range(n):
        m.append(list(map(int,input().split())))
    l=[]
    for i in range(x):
        l.append([])
    for i in m:
        for j in range(len(i)):
            l[j].append(i[j])
    for i in l:
        s += sumPairs(sorted(i),len(i))
    print(s)

File: Contests/Contest861/Q2.cpp
#include <bits/stdc++.h>
using namespace std;

#define int long long 
#define endl "\n"
#define file_io freopen("input.txt","r",stdin);freopen("output.txt","w",stdout)
#define fastio ios::sync_with_stdio(0);cin.tie(0);cout.tie(0)
#define pb push_back
#define PI 3.14159
#define pii pair<int,int>
map<int, int> mp;

int func(vector<int>& arr, int n) {
    int sum = 0;
    for (int i = n - 1; i >= 0; i--) {
        sum += i * arr[i] - (n - 1 - i) * arr[i];
    }
    return sum;
}

int32_t main() {
    fastio
    int num_of_test;
    cin >> num_of_test;
    for (int t = 0; t < num_of_test; t++) {
        int n, x;
        cin >> n >> x;
        vector<vector<int>> m(n, vector<int>(x));
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < x; j++) {
                cin >> m[i][j];
            }
        }
        vector<vector<int>> l(x);
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < x; j++) {
                l[j].pb(m[i][j]);
            }
        }
        int ans = 0;
        for (auto i : l) {
            ans += func(std::sort(i), i.size());
        }
        cout << ans << endl;
    }
}
